README.md

```diff
@@ -1 +1,13 @@
-# echo-server
\ No newline at end of file
+# http-server
+
+## To Run the App:
+
+Ensure you are in the correct folder - `/echo-server/EchoServer`
+
+Type `dotnet run` in your command line prompt
+
+## To Run the Tests:
+
+Ensure you are in the correct folder `/echo-server/EchoServer.Tests`
+
+Type `dotnet test` in your command line prompt
```

Comment:
This README appears to be missing sections to me.  I would expect a description, and also a setup section that would explain what `dotnet` even is, and either point to an installation guide or explain how to install it.  Also, if its important, what version to use.